content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -46,73 +47,61 @@ monitoring statistics by default:
+ logging, reporting, and visualizing log data.
+
+* [Google Stackdriver](https://cloud.google.com/stackdriver/) is a hosted monitoring service you can use to
+visualize and alert on important metrics in your application. 
+
+* [Prometheus](https://prometheus.io) can natively monitor kubernetes, nodes, and 
+  prometheus itself. The [Prometheus 
+  Operator](https://coreos.com/operators/prometheus/docs/latest/) simplifies 
+  Prometheus setup on Kubernetes, and allows you to serve the custom metrics API 
+  using the [Prometheus 
+  adapter](https://github.com/directxman12/k8s-prometheus-adapter). Prometheus 
+  provides a robust query language and a built-in dashboard for querying and 
+  visualizing your data. Prometheus is also a supported data source for 
+  [Grafana](https://prometheus.io/docs/visualization/grafana/).
+
+* [Sysdig](http://sysdig.com) provides full spectrum container and platform 
+  intelligence, and is a true container native solution. Sysdig pulls together 
+  data from system calls, Kubernetes events, Prometheus metrics, statsD, JMX, and 
+  more into a single pane that gives you a comprehensive picture of your 
+  environment. Sysdig also provides an API to query for providing robust and 
+  customizable solutions. Sysdig is built on Open Source. [Sysdig and Sysdig 
+  Inspect](https://sysdig.com/opensource/inspect/) give you the ability to freely 
+  perform troubleshooting, performance analysis and forensics. 
 
 ## CronJob monitoring
 
-### Kubernetes Job Monitor
-
-With the [Kubernetes Job Monitor](https://github.com/pietervogelaar/kubernetes-job-monitor) dashboard a Cluster Administrator can see which jobs are running and view the status of completed jobs.
-
-### New Relic Kubernetes monitoring integration
-
-[New Relic Kubernetes](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kubernetes-monitoring-integration) integration provides increased visibility into the performance of your Kubernetes environment. New Relic's Kubernetes integration instruments the container orchestration layer by reporting metrics from Kubernetes objects. The integration gives you insight into your Kubernetes nodes, namespaces, deployments, replica sets, pods, and containers.
+* With the [Kubernetes Job Monitor](https://github.com/pietervogelaar/kubernetes-job-monitor) dashboard, a Cluster Administrator can see which
+ jobs are running and view the status of completed jobs.
 
-Marquee capabilities:
-View your data in pre-built dashboards for immediate insight into your Kubernetes environment.
-Create your own custom queries and charts in Insights from automatically reported data.
-Create alert conditions on Kubernetes data.
-Learn more on this [page](https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kubernetes-monitoring-integration).
+* [New Relic Kubernetes](https://newrelic.com/platform/kubernetes) monitors
```

Comment:
Per #15576, this content no longer meets minimum standards. Remove L103-105.

Reply:
removed